modules/ROOT/pages/customsidebar.adoc

```diff
@@ -100,8 +104,12 @@ tinymce.PluginManager.add('myplugin', (editor) => {
     tooltip: 'My sidebar',
     icon: 'comment',
     onSetup: (api) => {
-      console.log('Render panel', api.element());
-    },
+     console.log('Render panel', api.element());
+     api.element().focus();
+     return () => {
+       editor.focus();
+     };
+   },
```

Comment:
```suggestion
      console.log('Render panel', api.element());
      api.element().focus();
      return () => {
        editor.focus();
      };
    },
```
Sorry Brian, I must have messed up the indentation by a single space, this should fix it.